index.md

```diff
@@ -4,49 +4,210 @@ title: Software Engineering for Busy Experts
+
+This seminar can be tweaked to use the language of your choice, or cover testing examples that resemble the kind of code you write to make this easier to apply. [Get in touch](mailto:hi@tkiley.co.uk) today to start writing automated tests today.
 
 ### "Good" Code
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec laoreet magna justo, eleifend sagittis magna pellentesque a. Vivamus nec porttitor tellus. Sed vitae volutpat diam. Nunc blandit mauris quis arcu pretium, sit amet hendrerit sapien molestie. Integer lobortis urna nec dignissim varius. Pellentesque at nunc vitae velit varius sodales at tincidunt elit. Vivamus mi sapien, tempor id lectus porta, laoreet feugiat eros. Cras vel dapibus risus, ut commodo ex. Nulla in auctor elit. Donec non placerat tellus, vitae pulvinar est. Ut eget tincidunt nisl. Curabitur congue libero eget tortor congue, laoreet gravida sem iaculis.
+Writing code is relatively easy, but writing "good" code is
+elusive. By following industry approaches over time the quality of the code will improve.
+
+**Code is a communication tool, writing good code will minimize misunderstanding between past and future developers.**
+
+My experience has taught me that good code is code that is:
+
+1. Readable
+2. Testable
+3. Maintainable
+
+This seminar will justify why writing good code is important, before providing a _large toolkit of guidelines and techniques_ that can be applied to your code to make it more readable, testable and maintainable.
+
+This seminar can be adjusted by reviewing some of your existing code to see the kind of problems that your experiencing and focusing on the techniques that will apply to your code base. [Get in touch](mailto:hi@tkiley.co.uk) today to start learning how to write better code.
+
+#### Seminar Syllabus
+
+The seminar can cover:
+
+ * A working definition of good code.
+ * How to make code readable.
+ * How to make code testable.
+ * How to make code maintainable.
+
+ Software & Technologies:
+
+* [Python](https://www.python.org/){:target="_blank"} - a popular programming language
 
 ### Code Review
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec laoreet magna justo, eleifend sagittis magna pellentesque a. Vivamus nec porttitor tellus. Sed vitae volutpat diam. Nunc blandit mauris quis arcu pretium, sit amet hendrerit sapien molestie. Integer lobortis urna nec dignissim varius. Pellentesque at nunc vitae velit varius sodales at tincidunt elit. Vivamus mi sapien, tempor id lectus porta, laoreet feugiat eros. Cras vel dapibus risus, ut commodo ex. Nulla in auctor elit. Donec non placerat tellus, vitae pulvinar est. Ut eget tincidunt nisl. Curabitur congue libero eget tortor congue, laoreet gravida sem iaculis.
+A code review process involves having one developer review another developers changes to the source code before it is submitted. Doing this effectively can spot defects and share knowledge.
```

Comment:
Flip to put the "why" bit first (and missing apostrophe):

"A code review process can spot defects and share knowledge. It involves having one developer review another developer's changes to the source code before it is submitted."